jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdConfiguration.java

```diff
@@ -59,7 +61,7 @@
      */
     public OpenIdConfiguration(String provider, String clientId, String clientSecret)
     {
-        this(provider, null, null, clientId, clientSecret);
+        this(provider, null, null, clientId, clientSecret, newHttpClient());
```

Comment:
probably better to pass null as the client and have the handling for null in the other constructor.